app/build.gradle

```diff
@@ -236,10 +242,9 @@ dependencies {
     androidTestCompile 'com.android.support.test.espresso:espresso-core:2.2.1'
     androidTestCompile 'com.android.support.test.espresso:espresso-intents:2.2.1'
     testCompile 'junit:junit:4.12'
-    testCompile "org.jetbrains.kotlin:kotlin-stdlib:1.0.6"
     testCompile "org.jetbrains.kotlin:kotlin-test-junit:1.0.6"
-    testCompile 'com.taroid.knit:knit:0.1.2'
-    testCompile 'com.github.sys1yagi:kmockito:0.1.2'
+    testCompile('com.taroid.knit:knit:0.1.2') { exclude module:"kotlin-stdlib" }
+    testCompile('com.github.sys1yagi:kmockito:0.1.2') { exclude module:"kotlin-stdlib" }
```

Comment:
Use resolution strategy, just like as `support-annotations` defined above the `dependencies`.